clang_tidy/clang_tidy_test.bzl

```diff
@@ -0,0 +1,119 @@
+"""A test rule to run clang-tidy"""
```

Comment:
Please note somewhere that "bash" is required in the env.

Reply:
done!